code/Player/Player.cs

```diff
@@ -98,7 +97,13 @@ public void Respawn()
 		Inventory.Clear();
 		Inventory.GiveDefaultLoadout();
 
-		Color = !Client.IsBot ? SelectedColor : Random.Shared.FromList( ColorPresets );
+		if ( Color == DefaultColor )

```

Comment:
I feel like we should move this logic somewhere else, got any ideas? Maybe using `GrubsEvent.Game.Start`?